app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -124,13 +128,15 @@ class HomeActivityTest {
 
   private val internalProfileId: Int = 1
   private lateinit var oppiaClock: OppiaClock
+  private lateinit var profileId: ProfileId
 
   @Before
   fun setUp() {
     Intents.init()
     setUpTestApplicationComponent()
     testCoroutineDispatchers.registerIdlingResource()
     profileTestHelper.initializeProfiles()
+    profileId = ProfileId.newBuilder().setInternalId(internalProfileId).build()
```

Comment:
out of 36 tests, there are only 10 which need this, so I was thinking if we can do this initialization there only. 

something like
```kotlin
    storyProgressTestHelper.markRecentlyPlayedForFractionsStory0Exploration0(
      profileId = ProfileId.newBuilder().setInternalId(internalProfileId).build(),
      timestampOlderThanAWeek = false
    )
    storyProgressTestHelper.markRecentlyPlayedForRatiosStory0Exploration0(
      profileId = ProfileId.newBuilder().setInternalId(internalProfileId).build(),
      timestampOlderThanAWeek = true
    )
```

Reply:
It is used in many places.

Reply:
> 1. add `ComingSoonTopicListViewModel` `ComingSoonTopicsViewModel` to https://github.com/oppia/oppia-android/blob/2511715a4770cda65df593cf6821b17b7f8f3d28/app/BUILD.bazel#L146
> 2. add `ComingSoonTopicsListView` to https://github.com/oppia/oppia-android/blob/2511715a4770cda65df593cf6821b17b7f8f3d28/app/BUILD.bazel#L309
> 3. Tests are failing, let me know if we can fix it together and save time as the number of test cases is big.
> 
> ![Screenshot 2020-12-31 at 10 10 06](https://user-images.githubusercontent.com/25057618/103394755-6e01a300-4b50-11eb-94bf-ed51a2870541.png)
> ![Screenshot 2020-12-31 at 10 09 21](https://user-images.githubusercontent.com/25057618/103394760-72c65700-4b50-11eb-917d-70d955b9e5b5.png)
> ![Screenshot 2020-12-31 at 10 06 11](https://user-images.githubusercontent.com/25057618/103394764-75c14780-4b50-11eb-9e21-909cfcc791ed.png)
> ![Screenshot 2020-12-31 at 10 02 13](https://user-images.githubusercontent.com/25057618/103394766-778b0b00-4b50-11eb-9c99-cbcbe5d99c8a.png)
> ![Screenshot 2020-12-31 at 09 53 06](https://user-images.githubusercontent.com/25057618/103394767-78bc3800-4b50-11eb-8ffd-84e8fa21e2d2.png)
> ![Screenshot 2020-12-31 at 09 46 03](https://user-images.githubusercontent.com/25057618/103394768-79ed6500-4b50-11eb-85fc-d3da59d1e83d.png)

Added viewmodels in Bazel.

All testcases are passing. PTAL
![Screenshot (1)](https://user-images.githubusercontent.com/16301028/103402738-a1a1f480-4b73-11eb-9746-90917cdeb36a.png)